universal-login-sdk/lib/api/DeployedWallet.ts

```diff
@@ -95,7 +92,7 @@ export class DeployedWallet implements ApplicationWallet {
     return this.sdk.getGasModes();
   }
 
-  async generateBackupCodes(): Promise<BackupCodesWithExecution> {
+  async generateBackupCodes(transactionDetails: Partial<Message>): Promise<BackupCodesWithExecution> {
```

Comment:
`Partial<Message>` -> `PaymentOptions`

Reply:
Refactoring in: https://github.com/UniversalLogin/UniversalLoginSDK/pull/1516